examples/stm32/xively_demo_mt/Src/httpserver-socket.c

```diff
@@ -0,0 +1,312 @@
+/**

```

Comment:
good point. I've created a story for this:
https://jira.3amlabs.net/browse/XCL-2723
Reason I've created story for this is that the STM32 project is far away from its final state, and this issue is not the last one to solve. And this sprint is not primarily about STM32. Additionally I've created this PR only to save the state of the STM32 project into development branch.
Although this latter is not mandatory. So we can even leave STM32 state here on this branch without merging to development branch.
What do you think, does it worth to put this into development branch now?